hathor/wallet/resources/thin_wallet/send_tokens.py

```diff
@@ -155,7 +155,8 @@ def _render_POST_stratum(self, context: _Context) -> None:
 
         # When using stratum to solve pow, we already set timestamp and parents
         stratum_deferred: Deferred[None] = Deferred()
-        stratum_deferred.addCallback(self._stratum_deferred_resolve, request)
+        # FIXME: Skipping mypy on the line below for now, as it looks like it's wrong but we don't have tests for it.
+        stratum_deferred.addCallback(self._stratum_deferred_resolve, request)  # type: ignore
```

Comment:
What's mypy error on this line?

Reply:
I removed this from this PR and it seemed to work on the current version.

Reply:
The `_stratum_deferred_resolve` callback expects to receive a `_Context` as argument, but in `addCallback` it is set with a `Request` instead. I think this is actually incorrect and exceptions would be raised, but this is never used and will probably be removed soon.

Reply:
I agree. Thanks!